src/ray/core_worker/transport/direct_actor_transport.cc

```diff
@@ -371,4 +380,18 @@ void CoreWorkerDirectTaskReceiver::HandleDirectActorCallArgWaitComplete(
   send_reply_callback(Status::OK(), nullptr, nullptr);
 }
 
+void CoreWorkerDirectTaskReceiver::CompleteAcceptRequest(
```

Comment:
It's somewhat ugly that we have two layers of saving the callbacks here. Can we combine them into one?

Reply:
I agree. I will just to pass `on_accept_callback_completion` to `executeTask` and build only one callback with this saving callback layer.